jobs/uaa-customized/templates/pre-start

```diff
@@ -12,18 +12,18 @@ JOB_DIR=/var/vcap/jobs/uaa-customized
 # Allow for > 128bit encryption
 cp -a "${PACKAGES_DIR}"/jce-policy/UnlimitedJCEPolicyJDK8/*.jar "${PACKAGES_DIR}/uaa/jdk/jre/lib/security"
 
-WARFILE="${PACKAGES_DIR}/uaa/tomcat/webapps/ROOT.war"
-JARFILE="WEB-INF/lib/cloudfoundry-identity-server-*.jar"
+export WARFILE="${PACKAGES_DIR}/uaa/tomcat/webapps/ROOT.war"
+JARFILE=$(unzip -v "${WARFILE}" | grep cloudfoundry-identity-server- | awk '{print $8}')
 
 WAR_TEMPDIR="$(mktemp -d)"
 pushd "$WAR_TEMPDIR"
 unzip "$WARFILE" "$JARFILE"
 mkdir -p templates/web
-cp -a "$JOB_DIR"/<%= p("template-directory") %>/web/* templates/web
-zip -u "$JARFILE" templates/web/*
+cp -a "$JOB_DIR"/<%= p("uaa-customized.template-directory") %>/web/* templates/web
+zip -r -u "$JARFILE" templates/web
 rm -rf templates
 zip -u "$WARFILE" "$JARFILE"
-unzip "$WARFILE" "resources/oss/*"
-cp -a "$JOB_DIR"/<%= p("template-directory") %>/resources/* resources/oss/
-zip -u "$WARFILE" resources/oss/*
+unzip "$WARFILE" "resources/*"
+cp -a "$JOB_DIR"/<%= p("uaa-customized.template-directory") %>/resources/* resources/
+zip -r -u "$WARFILE" resources
```

Comment:
We should probably clean up after ourselves here and `rm -rf resources` after the zip